scipy/stats/morestats.py

```diff
@@ -2090,6 +2090,21 @@ def ansari(x, y):
     ----------
     x, y : array_like
         Arrays of sample data.
+    alternative : {'two-sided', 'less', 'greater'}, optional
+        Defines the alternative hypothesis.
+        The following options are available (default is 'two-sided'):
+
+          * 'two-sided': two-sided, see explanation in Notes
+          * 'less': one-sided, see explanation in Notes
+          * 'greater': one-sided, see explanation in Notes
```

Comment:
```suggestion
        Default is 'two-sided'. See explanation in Notes.

```
Since there's not much information here, what do you think about condensing it?

Reply:
Done.